avocado/core/app.py

```diff
@@ -19,6 +19,7 @@
 import os
 import signal
 import sys
+import warnings
```

Comment:
This module is not being used... left over I assume?

Reply:
Yes, I will fix that on a v2. Also we have to supress the warning messages during the tests, to not messup with the outputs.